pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -92,6 +93,9 @@ in stdenv.mkDerivation {
   buildCommand = let
     browserBinary = "${chromium.browser}/libexec/chromium/chromium";
     getWrapperFlags = plugin: "$(< \"${plugin}/nix-support/wrapper-flags\")";
+    libPath = stdenv.lib.makeLibraryPath (
+      versionAtLeast version "72" && stdenv.lib.optional VAAPISupport libva
```

Comment:
Can we have an assert that `VAAPISupport` is only allowed `true` when version is > 72 instead? It seems weird to let the user specify VAAPISupport and then ignore it everywhere by `&&` with a version check.

Reply:
Might be cleaner indeed.

Reply:
Done